industrialucn/optimization/test_econstraint.py

```diff
@@ -0,0 +1,290 @@
+import logging
+import sys
+import unittest
+
+import numpy as np
+
+from industrialucn.optimization.econstraint import all_weakly_non_dominated, all_non_dominated
+from . import econstraint as ec
```

Comment:
It could be convenient to use a single import.

Reply:
You are right!
I made the changes here 09737a2a0c9ea40df76edf6b716a6fe85252044e